docs/understanding-tracking-design/organize-data-sources-with-source-applications/index.md

```diff
@@ -6,6 +6,16 @@ sidebar_label: "🆕 Source Applications"
 
 For data collection, you will often have different sources of information that correspond to applications designed for a particular purpose. These are what we will refer to as Source Applications.
 
+:::tip
+A guideline, which will address your needs most of the times, is to think of a Source Application as an independently deployable application system.
```

Comment:
"most of the times" → "most of the time"
"This will let you best manage" → "This approach will help you better manage"